common/src/main/scala/explore/components/TileController.scala

```diff
@@ -104,9 +106,13 @@ object TileController {
         getBreakpointFromWidth(p.layoutMap.map { case (x, (w, _, _)) => x -> w }, p.gridWidth)
       }
       // Store the current layout
-      .useStateViewWithReuseBy((p, _, _) => updateResizableState(p.layoutMap))
+      .useStateViewBy((p, _, _) => updateResizableState(p.layoutMap))
+      // Update the current layout if it changes upstream
+      .useEffectWithDepsBy((p, _, _, _) => p.layoutMap)((_, _, _, current) =>
```

Comment:
This is the core of the fix.

Reply:
Thanks. I wonder if we should implement a custom hook that implements the `useState`/`useEffectWithDeps` pattern.